### PR TITLE
[CCXDEV-12995] Add remote configurations for Rapid recommendations

### PR DESCRIPTION
**Pull request by JiriPapousek**
PR extends build script to build multiple configurations from `remote_configurations` folder. Also, structure of `build` directory has changed (as they are now different configurations for original endpoint and new v2 endpoint).

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,8 +1,25 @@
 #!/bin/bash
 
 mkdir -p ./build
```
Comment (juandspy) [inline build.sh]: this line can be deleted as the `-p` in the next lines already creates the parent folders

Comment (JiriPapousek) [reply build.sh]: Line was removed.

File: build.sh [hunk 2]
```diff
@@ -1,8 +1,44 @@
 #!/bin/bash
 
-mkdir -p ./build
 
-# Build the conditions
+TEMP=$(getopt -o o: --long configs:,conditions:,output: \
```
Comment (juandspy) [inline build.sh:6]: what is this? 😬

File: README.md [hunk 3]
```diff
@@ -26,6 +26,12 @@ Just execute the script:
 It will build the container image with the rules.
 
 
+## Testing
+
+`build.sh` script is tested using a Bats framework. In order to use them, you first need to download necessary dependencies by running `git submodule update --init --recursive`.
+
+Then you can run the tests with `test/test_helper/bats/bin/bats test/test.bats` from root directory of this repository.
```
Comment (juandspy) [inline README.md:33]: shall we add this to the CI?

File: build.sh (hunk 2 above)
Comment (JiriPapousek) [reply build.sh:6]: `getopt` was apparently necessary when I wanted to add arguments with long names to the script. The arguments themselves were necessary to change set of data for testing.

I can either add some comment to explain this, or remove CLI options and use something else, like env variables.

**Review by juandspy — APPROVED** [review]

File: build.sh (hunk 2 above)
Comment (juandspy) [reply build.sh:6]: just add a comment :) TY!

File: README.md (hunk 3 above)
Comment (JiriPapousek) [reply README.md:33]: CI step has been added

File: build.sh (hunk 2 above)
Comment (JiriPapousek) [reply build.sh:6]: Added :)

**Pull request merged**